File: java/src/main/java/com/experian/dvs/client/server/address/layout/RestApiElementSeparator.java
package com.experian.dvs.client.server.address.layout;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.Map;

public class RestApiElementSeparator {

    @JsonProperty("default")
    private String defaultSeparator;

    @JsonProperty("configuration_by_element")
    private Map<String, Object> configurationByElement;

    public String getDefaultSeparator() {
        return defaultSeparator;
    }

    public void setDefaultSeparator(String defaultSeparator) {
        this.defaultSeparator = defaultSeparator;
    }

    public Map<String, Object> getConfigurationByElement() {
        return configurationByElement;
    }

    public void setConfigurationByElement(Map<String, Object> configurationByElement) {
        this.configurationByElement = configurationByElement;
    }
}

File: csharp/DVSClient/DVSClient/server/address/layout/RestApiElementSeparator.cs
using Newtonsoft.Json;

namespace DVSClient.Server.Address.Layout
{
    public class RestApiElementSeparator
    {
        [JsonProperty("default")]
        public string? DefaultSeparator { get; set; }

        [JsonProperty("configuration_by_element")]
        public IDictionary<string, object>? ConfigurationByElement { get; set; }
    }
}
